src/index.html

```diff
@@ -1,17 +1,139 @@
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label class="label">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+          >
+        </label>
+        <label class="label">
+          Name:
+          <input
+            type="text"
+            name="name"
+            autocomplete="off"
+          >
+        </label>
+        <label class="label">
+          How old are You?
+          <input
+            type="number"
+            name="old"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+        <label class="label">
+          Full date of birth:
+          <input type="date" name="birth">
+        </label>
+        <label class="label">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="accept"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+        <label class="label">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+        <label class="label">
+          Password:
+          <input
+            type="password"
+            name="password"
+            minlength="5"
+            maxlength="30"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+        <label class="label">
+          Do you love cats?
+          <input
+            type="radio"
+            name="cats"
+            value="y"
+          >
+          Yes
+          <input
+            type="radio"
+            name="cats"
+            value="n"
+          >
+          No
+        </label>
+        <label class="label">
+          What is your favorite color?
+          <input type="color" name="color">
+        </label>
+        <label class="label">
+          What time do you go to bed?
+          <input type="time" name="time">
+        </label>
+        <label class="label">
+          What are your favorite brand of cars?
+          <select name="cars" multiple>
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+        <label class="label">
+          How do you rate our work?
+          <input
+            type="range"
+            name="rate"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend class="legend">Additional info:</legend>
```

Comment:
```suggestion
        <legend>Additional info:</legend>
```
you don't need class if you won't use it